specification/playwrighttesting/data-plane/Microsoft.PlaywrightTesting.AuthManager/stable/2024-12-01/examples/Accounts_Get.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "Accounts_Get",
+  "operationId": "Accounts_Get",
+  "parameters": {
+    "api-version": "2024-12-01",
+    "accountId": "sampleAccountId"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "sampleAccountId",
+        "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/dummyrg/providers/Microsoft.AzurePlaywrightService/accounts/myPlaywrightAccount",
+        "name": "myPlaywrightAccount",
+        "state": "Active",
+        "subscriptionId": "00000000-0000-0000-0000-000000000000",
+        "subscriptionState": "Registered",
+        "tenantId": "00000000-0000-0000-0000-000000000000",
+        "location": "westus",
+        "regionalAffinity": "Enabled",
+        "scalableExecution": "Enabled",
+        "reporting": "Disabled",
+        "localAuth": "Enabled"
```

Comment:
NIT: we can default value for reporting and localAuth

Reply:
would not impact as calls hit directly our service